low-level/framework/utils/sysfs_interface.py

```diff
@@ -23,19 +23,21 @@
 from pathlib import Path
 
 from framework.utils.utility import Utility
+from framework.utils.conf_utils import SSPL_CONF
+from cortx.utils.conf_store import Conf
```

Comment:
These two lines can be simplified as conf_utils already imported Conf.
`from framework.utils.conf_utils import SSPL_CONF, Conf`

Reply:
Ack. done the change.